docs/make.jl

```diff
@@ -0,0 +1,18 @@
+using Documenter
+using HSL
+
+makedocs(
+  modules = [HSL],
+  assets = ["assets/style.css"],
+  format = :html,
+  sitename = "HSL.jl",
+  pages = Any["Reference" => "reference.md"]
+)
+
+deploydocs(deps = nothing, make = nothing,
+  repo = "github.com/JuliaSmoothOptimizers/HSL.jl.git",
+  deps = Deps.pip("pygments", "mkdocs", "python-markdown-math"),
```

Comment:
When using `format = :html`, `mkdocs` are not called [(link)](https://juliadocs.github.io/Documenter.jl/stable/man/guide/#Output-formats-1). So the `deps` are not required.